### ISSUE TITLE
Adding readonly option

### ISSUE THREAD

**Issue by blue0513** [vk-lab/react-month-picker-input#13]
It's so cool react module!

One more thing I need is `readonly` option.  
In some cases, users manually input the date in `input field` but it's not proper values for the developer.

So, like bellow, I need `readonly` option to prevent users from inputting the date manually.

```js
<MonthPickerInput
  readonly
  year={2000}
  month={12}
  onChange={(_, year, month) => {
    console.log('year', year);
    console.log('month', month);
  }}
/>
```

**Comment by slavakisel** [comment]
@blue0513 Hello and thank you for your suggestion! Do I get it right, that in your case you want to use calendar only?

**Comment by slavakisel** [comment]
@blue0513 I've just released `1.2.0` version. It allows to set `mode` prop to one of `normal`, `readOnly` and `calendarOnly`. Please check it out, `calendarOnly` option is probably what you are looking for

**Comment by blue0513** [comment]
@slavakisel Thx for your quick and great response!!
Oh, this `calendarOnly` option is what I really needed! 😆 

Thank you very much, I'll use this released version 👍
